app/Actions/Album/Photos.php

```diff
@@ -67,12 +56,6 @@ public function get(Album $album): array
 		foreach ($photos as $photo_model) {
 			// Turn data from the database into a front-end friendly format
 			$photo = $photo_model->toReturnArray();
-			$photo['license'] = $photo_model->get_license($album->get_license());
```

Comment:
I don't know the answer to this question. But it cannot have been an optimization for speed, because it does not matter if you call `get_license` at this point or inside `toReturnArray` where it actually belongs.

Reply:
I am pretty sure this was an optimization with regard to the number of query to the album.